.github/workflows/build.yaml

```diff
@@ -22,15 +22,15 @@ jobs:
           fetch-depth: 0
 
       - name: Set up Python
-        uses: actions/setup-python@v3
+        uses: actions/setup-python@v4
         with:
           python-version: ${{ matrix.python-version }}
           cache: "pip"
           cache-dependency-path: "setup.cfg"
 
       - name: Build and install
         run: |
-          python -m pip install --upgrade pip "setuptools<65" "flake8<5"
+          python -m pip install --upgrade pip setuptools "flake8<5"
```

Comment:
I think you need to also make similar changes to cmake.yaml. (the python@v4 and setuptools)

Reply:
Updated `cmake.yaml` as well